pkgs/tools/misc/brotab/default.nix

```diff
@@ -18,15 +18,17 @@ python.pkgs.buildPythonApplication rec {
     setuptools
   ];
 
-  checkBuildInputs = with python.pkgs; [
-    pytest
-  ];
-
-  # test_integration.py requires Chrome browser session
-  checkPhase = ''
-    ${python.interpreter} -m unittest brotab/tests/test_{brotab,utils}.py
+  patchPhase = ''
+    substituteInPlace requirements/base.txt \
```

Comment:
Could we ask upstream about why they pin the dependencies?

Reply:
I wouldn't expect that we change their mind.

Reply:
Let's see what they'll say at:

https://github.com/balta2ar/brotab/commit/71b1c11e041a580a275a340e9e6ed0cb29e17420#commitcomment-82457696

Reply:
I am going to merge this PR with leaving this open because it is very common for python packages and accepted to do.